database.py

```diff
@@ -1,11 +1,15 @@
 """handles database session and data-type across app."""
+from __future__ import annotations
```

Comment:
I see no use of this import? Can you point out where? PRobably overlooking it.

Reply:
https://www.python.org/dev/peps/pep-0563/#id6

This is used to use classes which have not been completely defined but we still want to use them in annotations.

"DeclEnum" L96 function definition is an example of the same.